src/hlstrafe.cpp

```diff
@@ -503,6 +503,9 @@ namespace HLStrafe
 		if (curState.TargetYawOverrideActive)
 			return std::optional(static_cast<uint16_t>(curState.TargetYawOverride * M_INVU_DEG));
 
+		if (curState.ChangeTargetYawOffsetValue != 0.f && curState.ChangeTargetYawOffsetOver == 0.f)
+			return std::optional(static_cast<uint16_t>((vel_yaw + curState.ChangeTargetYawOffsetValue * M_DEG2RAD) * M_INVU_RAD));
```

Comment:
This should add to the values returned below. Right now you're essentially hardcoding velocity lock. Feel free to extract something into a separate function as necessary.